components/net_traits/image_cache_task.rs

```diff
@@ -86,21 +86,15 @@ pub fn load_image_data(url: Url, resource_task: ResourceTask, placeholder: &[u8]
                 return Ok(image_data);
             }
             ProgressMsg::Done(Err(..)) => {
-                // Failure to load the requested image will return the
```

Comment:
I added this extra logic to ensure that even if the resources folder was not found (or maybe the placeholder file is not present) we would still run fine. Did you tested what happens if the rippy.jpg file is not present (i.e. remove the file)?